releases/2020-11/js.md

```diff
@@ -34,9 +38,65 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 ## Release highlights
 
-### _Package name_
+---
+
+=== COPY THIS AND ADD THE INFORMATION OF YOUR PACKAGE: ===
+
+Keep in mind that:
+
+- Including the package name in the headers makes the URL links work for multiple packages.
+- The format of this file will be cleaned up once all of your proposals are in.
+
+---
+
+### _Project name_ 
+
+#### @azure/_package-name_@_version_ [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/<service-folder>/<package-folder>/CHANGELOG.md)
+
+(leave blank)
+
+##### Breaking Changes on @azure/_package-name_@_version_
+
+- _Add one or more, or remove the "Breaking Changes on ..." entire section._
+
+##### New Features on @azure/_package-name_@_version_
+
+- _Add one or more, or remove the "New Features on ..." section._
+
+##### Major Fixes on @azure/_package-name_@_version_
+
+- _Add one or more, or remove the "Major Fixes on ..." section._
+
```

Comment:
### Event Hubs 

#### @azure/event-hubs@3.5.1 [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/CHANGELOG.md)

##### Major Fixes on @azure/event-hubs@3.5.1

- Fixes an issue where the `processEvents` handler passed to `EventHubConsumerClient.subscribe()` could ignore the `maxWaitTimeInSeconds` parameter after a disconnection, resulting in `processEvents` being invoked rapidly without events.

Reply:
Added it here: https://github.com/azure-sdk/azure-sdk/pull/1/files#diff-98bb59edc392f52d470ae8cf57987df7ff203ff8b6747f840379fe1a33dcc41dR104